index.html

```diff
@@ -0,0 +1,142 @@
+                                        <label class="form-check-label" for="text">Text Entry</label>
+                                    </div>
+                                </div>
+                            </div>
+                        </fieldset>
+                        <div class="form-group mb-0 row align-items-center" id="file-group">
+                            <label for="code-file" class="col-sm-2 col-form-label"><strong>Input code as file</strong></label>
+                            <input type="file" name="code-file" id="code-file" class="form-control-file col-sm-10"/>
+                        </div>
+                        <div class="form-group row d-none" id="text-group">
+                            <label for="code-text" class="col-sm-2 col-form-label"><strong>Input code as text</strong></label>
+                            <div class="col-sm-10">
+                                <textarea name="code-text" id="code-text" class="form-control form-control-sm mb-1 " rows="10"></textarea>
+                            </div>
+                        </div>
+                        <div class="form-group row align-items-center">
+                            <label for="tab-size" class="col-sm-2 col-form-label"><strong>Tab Size</strong></label>
+                            <div class="container justify-content-start col">
+                                <input type="number" id="tab-size" name="tab-size" required="required" value="3" class="form-control col-6"/>
+                            </div>
+                        </div>
+                        <button type="submit" class="btn btn-primary mb-4" name="submit" id="submit" disabled>Analyze Code</button>
+                    </form>
+                </section>
+            </div>
+            <div class="container">
+                <h2 class="collapse-toggler" type="button" role="button" data-toggle="collapse" data-target="#lint-output">
+                    <img class="collapse-button closed" src="img/expand_more.svg" alt="collapse button"/>
+                    Errors Found
+                </h2>
+                <section class="collapse" id="lint-output">
+                    <div class="alert alert-warning alert-dismissible fade show" role="alert">
+                        This tool is meant to provide a lot of information, but makes no guarantees that what is found by this tool will encompass all possible style issues in the input code. In fact, while this tool is made to catch a lot of style issues, creating a tool to catch every possible style error is incredibly difficult and certainly not something I tried to do, so make sure to give your code a second look yourself.
+                    </div>
+                    <ul class="nav nav-tabs">
+                        <li class="nav-item">
+                            <a class="nav-link active" data-toggle="tab" href="#errors-highlighting">Error Highlighting</a>
```

Comment:
For this to be accessible by a screen reader or keyboard, it needs a valid href. Test with a keyboard and see if you do everything!